indra/newview/app_settings/settings.xml

```diff
@@ -9164,6 +9164,61 @@
     <key>Value</key>
     <integer>0</integer>
   </map>
+  <key>RenderScreenSpaceReflectionIterations</key>
+  <map>
+    <key>Comment</key>
+    <string>Renders screen space reflections to better account for dynamic objects with reflection probes.</string>
```

Comment:
Looks like these debug settings are using the same description.

Reply:
Yup - my bad.  Forgot to update the descriptions.  Gonna update those now.